src/demo-app/autocomplete/autocomplete-demo.ts

```diff
@@ -3,6 +3,16 @@ import {FormControl, NgModel} from '@angular/forms';
 import 'rxjs/add/operator/startWith';
 import 'rxjs/add/operator/map';
 
+interface State {
+  code: string;
+  name: string;
+}
+
+interface StateGroup {
```

Comment:
Failing presubmit because of TS error for "private name `StateGroup`" and "private name `State`". I think you need to export these interfaces.